src/xarray_regrid/regrid.py

```diff
@@ -125,26 +131,72 @@ def most_common(
 
         Args:
             ds_target_grid: Target grid dataset
-            time_dim: Name of the time dimension. Defaults to "time".
-            max_mem: (Approximate) maximum memory in bytes that the regridding routine
-                can use. Note that this is not the total memory consumption and does not
-                include the size of the final dataset. Defaults to 1e9 (1 GB).
+            expected_groups: Numpy array containing all labels expected to be in the
+                input data. For example, `np.array([0, 2, 4])`, if the data only
+                contains the values 0, 2 and 4.
+            time_dim: Name of the time dimension. Defaults to "time". Use `None` to
```

Comment:
Instead of specializing to time, how about setting `reduce_dim`. Where the default is set using `ds_target_grid`? and users can *add* `"time"` if they want.

Reply:
While I often create a target grid from scratch in examples, many users will have a dataset with a grid that they want other data to be regridded to. 

For example: projecting the CAMS CO2 dataset (0.75 deg) to ERA5 (0.25 deg). 
```py
ds_cams_regridded = ds_cams.regrid.linear(ds_era5)
```

Reply:
How do you pick latitdue, a longitude in that case? You could do the same here ...

Reply:
I have opened https://github.com/xarray-contrib/xarray-regrid/issues/47 to discuss this point